### PR TITLE
[Refactor] floatingNav CSS

### PR DESCRIPTION
**Pull request by imjuangarcia**

### CODE REVIEW

File: components/utilities/floatingNav.js [hunk 1]
```diff
@@ -112,11 +149,43 @@ const Headings = ({ headings, activeId }) => {
...
+      `}
+      key={`toc-${index}`}
+    >
+      <a
+        href={heading.target}
+        className={`
+          border-b-0
+          inline-block truncate w-40
+          leading-4
+          hover:opacity-70 hover:border-b-0 hover:no-underline
```
Comment (tvst) [inline components/utilities/floatingNav.js:183]: Move the above into a class

File: components/utilities/floatingNav.js [hunk 2]
```diff
@@ -112,11 +149,43 @@ const Headings = ({ headings, activeId }) => {
...
+        ${
+          heading.level === "H1" || heading.level === "H2"
+            ? "pl-6"
+            : heading.level === "H3"
+            ? "pl-8"
+            : heading.level === "H4"
+            ? "pl-10"
+            : heading.level === "H5"
+            ? "pl-12"
+            : "pl-14"
```
Comment (tvst) [inline components/utilities/floatingNav.js:172]: Move the above into a class.


For the heading picker, you could go different routes:

heading.level == "H1" ? styles.headingH1 : etc

styles[heading.level]

use "heading1" as a style
:global(heading1)

**Review by tvst — APPROVED** [review]
LGTM after changes

File: components/utilities/floatingNav.js (hunk 2 above)
Comment (imjuangarcia) [reply components/utilities/floatingNav.js]: Fixed on c8573ad

File: components/utilities/floatingNav.js (hunk 1 above)
Comment (imjuangarcia) [reply components/utilities/floatingNav.js]: Fixed on c8573ad

**Pull request merged**